lib/l10n/intl_en.arb

```diff
@@ -64,8 +67,10 @@
   "actionSignInWith": "Sign in with {provider}",
   "actionResetPassword": "Reset password",
   "actionSendEmail": "Send e-mail",
+  "actionConfirmChangeEmail": "Confirm change email",
```

Comment:
This looks kind of awkward, maybe call it simply "Change email" (and change the name too). It would also be consistent with the Delete account dialog.